### PR TITLE
Lps-168680: Remove input permission from UI after blog creation

### PR DESCRIPTION
**Pull request by kristofloneck**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
@kristofloneck please, resolve the SF issues

Thanks!

File: modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -1521,6 +1521,24 @@ public BlogsEntry updateStatus(
 				_pingTrackbacks(
 					entry, trackbacks, pingOldTrackbacks, serviceContext);
 			}
+
+			// Resources
+
+			if (GetterUtil.getBoolean(
+					serviceContext.getAttribute("showPermissions"))) {
```
Comment (adolfopa) [inline modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java:1528]: Where is this service context attribute initialized? Are you sure this is correct?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java (hunk 1 above)
Comment (kristofloneck) [reply modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java:1528]: It works. When I tried to make the check null safe the source formating reseted back to this form.

**Comment by kristofloneck** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
@kristofloneck Can you fix the sf issues?

**Comment by kristofloneck** [comment]
ci:test:sf

**Comment by kristofloneck** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/edit_entry.jsp [hunk 2]
```diff
@@ -336,6 +336,8 @@ renderResponse.setTitle(blogsEditEntryDisplayContext.getPageTitle(resourceBundle
 								modelName="<%= BlogsEntry.class.getName() %>"
 							/>
 						</aui:fieldset>
+
+						<aui:input name="showPermissions" type="hidden" value="<%= true %>" />
```
Comment (adolfopa) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/edit_entry.jsp:340]: Can you rename the parameter to `addEntryResources`. In the backend it is confusing (it is not showing anything).

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by kristofloneck** [comment]
ci:test:sf

**Comment by kristofloneck** [comment]
ci:test:sf

**Comment by kristofloneck** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
The above Poshi tests failure are not related to this PR.

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:reevaluate:1377401_7818

**Comment by Hanlf** [comment]
The above Poshi failures are not related to this PR.

**Comment by Hanlf** [comment]
Sent it to BChan here: https://github.com/brianchandotcom/liferay-portal/pull/128529

**Pull request closed without merging**